.github/workflows/build.yaml

```diff
@@ -5,41 +5,57 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      STARLORD_IMAGE_TAG: starlord.inscloudgate.net/deploy/muss
     steps:
+      # general setup
       - uses: actions/checkout@v2
-      - name: Unshallow
-        run: git fetch --prune --unshallow
+      - name: checkout private actions
+        uses: actions/checkout@v2
+        with:
+          repository: instructure-bridge/actions
+          token: ${{ secrets.GIT_HUB_TOKEN }}
+          path: .github/actions
+          ref: master
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
           go-version: 1.13
+      
+      # build and test
+      - name: Unshallow
+        run: git fetch --prune --unshallow
       - name: GoReleaser Build
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
           args: build --snapshot
       - name: Docker Build
-        run: docker build --tag ${{ github.sha }} .
+        run: docker build --tag ${STARLORD_IMAGE_TAG}:latest .
```

Comment:
don't we still want some sort of sha-based tag published?

Reply:
not for muss. it only uploaded to `latest` during a merge to master.